modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,3 +1,12 @@
+resource "aws_db_subnet_group" "database" {
+  name       = "${var.infrastructurename}-vpc"
+  subnet_ids = var.private_subnets
+
+  tags = {
+    Name = "My DB subnet group"

```

Comment:
please delete this tag.

Reply:
Well, we should have tags on our resources, it's a good practice and easier to track resources. I expected a comment on this, but more towards value (ie "what should we put as tag") rather than "do we need a tag here"?

Reply:
Done